gutenberg.php

```diff
@@ -181,3 +184,5 @@ function register_site_icon_url( $response ) {
 }
 
 add_filter( 'rest_index', 'register_site_icon_url' );
+
+add_theme_support( 'block-based-widgets' );
```

Comment:
The support is enabled by default in the Gutenberg plugin to bring visibility to the feature and get more testing, but in Core it could be made opt-in instead of opt-out.

Reply:
Can we rename it to something like "widgets-block-editor"?